### ISSUE TITLE
B_inv_source, B_inv_coord and B_inv_AFmax

### ISSUE THREAD

**Issue by lgmgeo** [lgmgeo/AnnotSV#223]
**Question from an AnnotSV user:**

I am writing to you because we have annotated our SV tables with AnnotSV and we are not sure of in which case you consider that a benign inversion completely overlaps the SV to annotate so that the columns B_inv_source, B_inv_coord and B_inv_AFmax get anotated. Apparently, coordinates of both do not match and it seems problematic to us consider an inversion benign because it is contained within a known benign inversion, since the known benign inversion may leave a gene intact while the contained one may cut it in some point. Could you explain a bit your approach?

Basically, we would like to know if it is reliable to use these columns to assume an inversion is benign when the columns show other benign inversions overlapping.

Thank you very much for any help,

**Comment by lgmgeo** [comment]
Thank you for sharing your point of view. Really interesting debate on how to rank/prioritize "inversions"!

I'm definitely not an expert on SV ranking, so the discussion is open.

- Actually, SV ranking is only available for GAIN (duplications) and LOSS (deletions). The provided pathogenic score is an adaptation of the work provided by the joint consensus recommendation of ACMG and ClinGen (Riggs et al., 2020). 

- Insertions and inversions are not yet ranked.

> it seems problematic to us consider an inversion benign because it is contained within a known benign inversion, since the known benign inversion may leave a gene intact while the contained one may cut it in some point. 

Absolutely, you're right.
The columns B_inv_source, B_inv_coord and B_inv_AFmax are present for informational purposes but do not allow an inversion to be classified as benign.

> is it reliable to use these columns to assume an inversion is benign when the columns show other benign inversions overlapping.

No.
I think you should look at:
- breakpoints of an inversion (knowing if the BNDs are located in a gene or not)
- overlapping genes or closest genes
Then, analyse these genes with the damaging probability provided for each gene (Exomiser and PhenoGenius modules)

**Comment by lgmgeo** [comment]
@jbmtp @thomasguignard @jmuller:
Any comments on how to classify inversions with AnnotSV annotations?

**Comment by thomasguignard** [comment]
I agree with @lgmgeo.
If the known benign inversion positions are extremely close to your inversion query, you might consider the information.

@lgmgeo, let's think about this improvement: AnnotSV could assume to apply a "pseudo-ACMG" class to INV/INS and others exotic SV.
Since biological interpretation of INV impact depends on wether the gene:
- is truncated 
- is OMIM morbid, with AD/AR inheritance mode, 
- has HI score = 3 or 2
- has high Exomiser/PhenoGenius score
It could be OK to consider it as probably pathogenic.

Another consideration about INV is related to its size. Big INVs may affect TAD structure/position and the affected genes expression.

**Comment by lgmgeo** [comment]
The most difficult thing will be to establish the right weightings for each criterion. We cannot rely on ACMG's work for inv/ins.
But yes, definitely, the need to propose a classification of inversions and insertions is real.
The most difficult thing is finding the time and what to test/evaluate this new scoring. Unfortunately, not in the near future for AnnotSV.

**Comment by ZaharaMC** [comment]
Thank you very much for you insights!

So then, would you say, following the same logic, that columns B_gain_source, B_gain_coord and B_gain_AFmax as well as P_gain_source, P_gain_coord and P_gain_AFmax are mostly informational? Because there are examples in which a pathogenic duplication can be contained within a benign one (see pic)

![Scheme_DUP](https://github.com/lgmgeo/AnnotSV/assets/54777492/00023ffe-06cd-4c5e-adaf-690df69b00e9)

However, I would consider B_loss_source, B_loss_coord and B_loss_AFmax as well as P_loss_source, P_loss_coord and P_loss_AFmax quite reliable predictors of a deletion being benign or pathogenic (it would be very rare to have a pathogenic deletion contained within a benign one for deletions larger than 50bp). Am I wrong?

Thanks again for any thoughts!

(I was not sure if I should keep this within this issue, hope it is fine)

**Comment by lgmgeo** [comment]
Actually, for gain and loss, we follow the ACMG / Clingen recommendation.
Please see https://lbgi.fr/AnnotSV/Documentation/Scoring_Criteria_AnnotSV_latest.xlsx

Of course, you're right, a pathogenic duplication can be contained within a benign one. 
Several sections are evaluated and each criterion contributes weight to the final score.

**Comment by ZaharaMC** [comment]
I see, so we should always refer to ACMG class. We were hoping to rank VUS (ACMG=3) looking at P_gain/loss columns, that is why we were trying to understand better the criteria to annotate these columns and the info they could give us.

Thanks!!
